cpp/tests/collect_list/collect_list_test.cu

```diff
@@ -0,0 +1,486 @@
+/*
```

Comment:
This test probably belongs in a `.cpp` file, instead of a `.cu`. I will check that renaming the file works.

Reply:
Done. It compiles faster, and points out unused variables more aggressively (thanks to `gcc`, vs `nvcc`).